src/Utilities/Digraph.hpp

```diff
@@ -3,10 +3,10 @@
 
 #pragma once
 
+#include <brigand/brigand.hpp>
```

Comment:
Thinking "out loud" here... I don't really love that this breaks how TMPL.hpp is the one place to hook into our TMP library (I do realize that it was already broken by the cyclic dependency in a hidden way).

A thought: what if we renamed Digraph.hpp to TmplDetails.hpp or similar, to make it more obvious that this is another place that plugs into the TMP library interface?

(And an even more tangential thought: shouldn't TMPL.hpp -> Tmpl.hpp to match how we CamelCase all other acronyms? not for this PR though.)

Reply:
How about `TmplDigraph.hpp`? I'm not sure what all would/should go into `TmplDetails.hpp` since that sounds more encompassing than just the digraph code.

As to the rename, yes, just haven't gotten around to it :)

Reply:
`TmplDigraph.hpp` sounds good to me :)